examples/EvaluationDataToApplicationInsights/Program.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.FeatureManagement.Telemetry.ApplicationInsights;
```

Comment:
Shouldn't this have been removed? The referenced namespace doesn't exist anymore.